tensorflow/python/autograph/operators/py_builtins.py

```diff
@@ -384,8 +384,29 @@ def _py_filter(function, iterable):
   return filter(function, iterable)
 
 
+def any_(iterable):
+  if isinstance(iterable, dataset_ops.DatasetV2):
+    return _tf_dataset_any(iterable)
+  return _py_any(iterable)
+
+
+# any() operation is essentially a "if first True element exist".
+# For that it could be translated to `filter(True)` to filter out
+# only `True` element, and then `take(1)`. This works in tf.data
+# as tf.data's filter+take is done in pipeline so it will stop
+# as soon as `take(1)` returns.
+def _tf_dataset_any(iterable):
+  return iterable.filter(
```

Comment:
The chaining causes a wrapping that's a bit awkward. Consider breaking it on multiple lines:

```
ds = iterable.filter(lambda x: x)
ds = ds.take(1)
... etc.
```

Reply:
Thanks @mdanatg, the chaining has been removed.